### ISSUE TITLE
More Meeting Option switches (skip pre-join page, disable Google Analytics)

### ISSUE THREAD

**Issue by adamzea** [timetheoretical/jitsi-meet-outlook#27]
Here are a couple more switches you might consider adding to the Meeting Options Settings.

`#config.analytics.disabled=true`
   - start with no Google analytics

`#config.prejoinPageEnabled=false`
   - skip pre-join dialogue page. There's a new pre-join page that lets you test the camera and enter your name before joining the meeting. Above will turn it off so the link goes straight to the meeting.

Those work the same way as your existing "start muted", "no video on start" options.

**Comment by timetheoretical** [comment]
I agree, these would be good features to include. Will look into adding these features.

As for the first one, I envisage this could be more of a permanent option that the user could specify in the settings dialogue. The second one could very well be implemented in the form of a button in the appointment view, as you point out.
